apps/ejabberd/src/ejabberd_admin.erl

```diff
@@ -155,6 +166,59 @@ commands() ->
 %%% Server management
 %%%
 
+-spec join_cluster(string()) -> {ok, string()} | {pang, string()} | {mnesia_error, string()} | {error, string()}.
+join_cluster(NodeString) ->
+    NodeAtom = list_to_atom(NodeString),
+    NodeList = mnesia:system_info(running_db_nodes),
+    case lists:member(NodeAtom, NodeList) of
+        true ->
+            String = io_lib:format("The node ~s has already joined the cluster~n", [NodeString]),
+            {ok, String};
+        _ ->
+            do_join_cluster(NodeAtom)
+    end.
+
+do_join_cluster(Node) ->
+    try mongoose_cluster:join(Node) of
+        ok ->
+            String = io_lib:format("You have successfully joined the node ~p to
+      the cluster with node member ~p~n", [node(), Node]),
```

Comment:
I must have split it because the line was too long but not having in mind that it's text to display!
I will fix it